playground/mel_playground.cppo.ml

```diff
@@ -186,14 +186,23 @@ let compile =
     let types_signature = ref [] in
     warnings_collected := [];
     try
+      let lexbuf = Lexing.from_string str in
+      Ocaml_common.Location.input_lexbuf := Some lexbuf;
+      Location.input_lexbuf := Some lexbuf;
```

Comment:
I saw these used in `Ast_io` too, is this something new that 5.2 requires? And does it affect somehow the older versions?

Reply:
This is related to https://github.com/ocaml/ocaml/pull/12991

Reply:
basically if we don't do this, tests won't include source code quotations.